independent-publisher-connectors/SOS Inventory/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2218 @@
+						"type": "string",
+						"x-ms-summary": "Location",
+						"description": "Inventory at given location",
+						"required": false
+					},
+					{
+						"in": "query",
+						"name": "archived",
+						"type": "string",
+						"x-ms-summary": "Archived",
+						"description": "Archived",
+						"enum": [
+							"Yes",
+							"No"
+						],
+						"required": false
+					},
+					{
+						"in": "query",
+						"name": "status",
+						"description": "Expired, Recalled, Neither",
+						"type": "string",
+						"x-ms-summary": "Status",
+						"enum": [
+							"expired",
+							"recalled",
+							"notexpirednotrecalled"
+						],
+						"required": false
+					},
+					{
+						"in": "query",
+						"name": "limit",
+						"x-ms-summary": "Limit",
+						"description": "Max records to return, default 10",
+						"type": "integer",
+						"default": 10
+					}
+				]
+			}
+		},
+		"/location": {
+			"get": {
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"count": {
+									"type": "integer",
+									"description": "# Locations Returned"
+								},
+								"totalCount": {
+									"type": "integer",
+									"description": "# Matching Locations"
+								},
+								"data": {
+									"type": "array",
+									"items": {
+										"type": "object",
+										"properties": {
+											"id": {
+												"description": "Location ID",
+												"type": "integer"
+											},
+											"syncToken": {
+												"description": "syncToken",
+												"type": "integer"
+											},
+											"name": {
+												"description": "Location Name",
+												"type": "string"
+											},
+											"binTracking": {
+												"description": "Bin Tracking",
+												"type": "boolean"
+											},
+											"bins": {
+												"type": "array",
+												"items": {
+													"type": "object",
+													"description": "Bins",
+													"properties": {
+														"id": {
+															"type": "integer",
+															"description": "ID"
+														},
+														"syncToken": {
+															"type": "integer",
+															"description": "syncToken"
+														},
+														"number": {
+															"type": "string",
+															"description": "Bin Number"
+														},
+														"location": {
+															"type": "string",
+															"description": "Location"
+														},
+														"sortOrder": {
+															"type": "integer",
+															"description": "sortOrder"
+														}
+													}
+												}
+											},
+											"address": {
+												"type": "object",
+												"description": "Address",
+												"properties": {
+													"line1": {
+														"type": "string",
+														"description": "line1"
+													},
+													"line2": {
+														"type": "string",
```

Comment:
@Harold-Anderson these descriptions for address are also considered non-translatable. You can change the value if you don't want to keep them as it is. E.g. "Line 1" or "First line".